.github/workflows/release-discord-notify.yml

```diff
@@ -0,0 +1,27 @@
+name: release-discord-notify
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  notify:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Discord Notify
+        uses: fjogeleit/http-request-action@v1
+        with:
+          url: ${{ secrets.OKP4_DISCORD_WEBHOOK }}
+          method: 'POST'
+          customHeaders: '{"Content-Type": "application/json"}'
+          data: '{
```

Comment:
Would be better to use the literal style (denoted by the `|` indicator) to have a correct interpretation of multiline strings.

Reply:
I think the `|` is only when string is not quoted

Reply:
I'd suggest to remove the single-quoted style in favor of the literal style. The single quote style is more limited, e.g. the `'` has to be doubled, while the literal style does not need special care.

Reply:
I tried the literal style, but it seems to fail the formating check on the Discord side.

Reply:
Also quoted multi-line seems to not work either.

Reply:
```
name: release-discord-notify

on:
  release:
    types: [published]
  workflow_dispatch:

jobs:
  notify:
    runs-on: ubuntu-22.04
    steps:
      - name: Notify Discord
        uses: fjogeleit/http-request-action@v1
        with:
          url: https://discord.com/api/webhooks/1017323602078597130/MxYZXO_mVul8QZbIjiFVe0ueWWLM5Qg6Y2W6Y1RZN6o5vMRMLiDK75OBDibImFr4UzDE
          method: 'POST'
          customHeaders: '{"Content-Type": "application/json"}'
          data: '{"avatar_url": "https://avatars.githubusercontent.com/u/98603954?v=4","username": "Bot Anik","content": "🚨 A new version of @${{github.repository}} ${{ github.event.tagName }} has been released! \n \n 👉 Changelog: https://github.com/${{github.repository}}/releases/tag/${{github.event.tagName}} \n 👉 Official repo: https://github.com/${{github.repository}}"}'
```

Reply:
This one works

Reply:
Please use a literal style it''s more readable and safer:

```yaml
name: release-discord-notify

on:
  release:
    types: [published]
  workflow_dispatch:

jobs:
  notify:
    runs-on: ubuntu-22.04
    steps:
      - name: Notify Discord
        uses: fjogeleit/http-request-action@v1
        with:
          url: https://discord.com/api/webhooks/1017323602078597130/MxYZXO_mVul8QZbIjiFVe0ueWWLM5Qg6Y2W6Y1RZN6o5vMRMLiDK75OBDibImFr4UzDE
          method: 'POST'
          customHeaders: '{"Content-Type": "application/json"}'
          data: |-
              {
                "avatar_url":"https://avatars.githubusercontent.com/u/98603954?v=4",
                "username":"Bot Anik",
                "content":"🚨 A new version of @${{github.repository}} ${{ github.event.tagName }} has been released! \n \n 👉 Changelog: https://github.com/${{github.repository}}/releases/tag/${{github.event.tagName}} \n 👉 Official repo: https://github.com/${{github.repository}}"
              }
```

Reply:
This one works, I think it's due to the missing `-` when I tried

Reply:
But I don't really love the fact that the webhook message is on one line.

Reply:
Have you an idea how can we split this ?

Reply:
Cause when I split this I get "Error: {"code":400,"message":{"_misc":["Only dictionaries may be used in a DictType"]}}"

Reply:
@warstrolo The JSON format doesn't support multiline

Reply:
that's sad